packages/wrangler/src/api/startDevWorker/LocalRuntimeController.ts

```diff
@@ -197,6 +197,7 @@ export class LocalRuntimeController extends RuntimeController {
 					userWorkerInnerUrlOverrides: {
 						protocol: data.config?.dev?.origin?.secure ? "https:" : "http:",
 						hostname: data.config?.dev?.origin?.hostname,
+						port: data.config?.dev?.origin?.hostname ? "" : undefined,
```

Comment:
This looks like a bug fix that deserves a changeset?

Reply:
Changeset added in https://github.com/cloudflare/workers-sdk/pull/6330/commits/9f1a6650b23e8c707209ec122adfa53256950c94